packages/main/src/MultiComboBoxGroupItem.ts

```diff
@@ -39,16 +18,18 @@ const metadata = {
  * @implements sap.ui.webc.main.IMultiComboBoxItem
  * @since 1.4.0
  */
-class MultiComboBoxGroupItem extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
-
-	static get dependencies() {
-		return [
-			GroupHeaderListItem,
```

Comment:
GroupHeaderListItem not needed?

Reply:
as @nnaydenow noticed for the ComboBox, the Group Header List Item is rendered in the Popover of the MCB, therefore I require it in the source of the MCB.